### PR TITLE
Feat/language 05

### PR DESCRIPTION
**Pull request by cmgustavo**
* Updates template.POT with new strings
* Updates spanish.po
* Fix logs page

Note: some strings that use native (cordova) notification can not be translated with gettext (for now).

### CODE REVIEW

File: po/template.pot [hunk 1]
```diff
@@ -196,10 +266,22 @@ msgstr ""
...
+#: src/js/controllers/importLegacy.js
+msgid "Failed to import wallets"
+msgstr ""
+
 #: public/views/create.html
 msgid "Family vacation funds"
 msgstr ""
 
+#: src/js/controllers/walletHome.js
+msgid "Fetching Payment Informantion"
```
Comment (Kirvx) [inline po/template.pot]: Information*

Comment (cmgustavo) [inline po/template.pot]: When you make a payment using BIP72 (Payment Protocol URI), Copay try to get information from the URL. So, this message will show to the user while Copay downloads information about the payment (address, amount, merchant, etc...).

Comment (cmgustavo) [inline po/template.pot]: There is a typo! Please, let me fix it and push again.

Comment (Kirvx) [inline po/template.pot]: My intervention was only for the typo error ^^

**Comment by cmgustavo** [comment]
Updated template.POT. This PR will be merged soon.

Ping @dabura667 and @Kirvx

**Comment by Kirvx** [comment]
Thanks :)
Done https://github.com/bitpay/copay/pull/2666

**Pull request merged**